ui-test/public-suite/extension-activity-test.ts

```diff
@@ -0,0 +1,59 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+import { SideBarView, ActivityBar, WebDriver, VSBrowser, EditorView } from 'vscode-extension-tester';
+import { expect } from 'chai';
+import { viewHasItems } from '../common/conditions';
+import { views } from '../common/constants';
+
+export function extensionActivityTest() : void{
+
+  describe('Tekton Activity Test', () => {
+    let driver: WebDriver;
+
+    before(async function() {
+      this.timeout(200000);
```

Comment:
Quite a timeout! Is it necessary? I would lower, in general, all timeouts to something like twice as much as it really takes.